api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -12,8 +12,13 @@ import "gogoproto/gogo.proto";
 // Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
 
 message RBAC {
-  // Specify the RBAC rules to be applied globally
-  config.rbac.v2alpha.RBAC rules = 1 [(validate.rules).message.required = true];
+  // Specify the enforcement mode RBAC rules to be applied globally.
+  // Disable the filter for enforcement mode if rules isn't set.
+  config.rbac.v2alpha.RBAC rules = 1;
```

Comment:
Before we remove "[(validate.rules).message.required = true]", @rodaine can you please review this part? I think you had it there on purpose. 

For RBAC disabled case, we can just leave the entire RBAC filter spec empty. RBAC filter will not be invoked in that case.

Reply:
See my other comment on the removal of `disabled`